packages/lib-user/src/components/shared/AllProjects/Projects/Projects.js

```diff
@@ -50,8 +50,8 @@ function Projects({
         </Box>
       ) : (
         <Box align='center'>
-          <StyledBox forwardedAs='ul' direction='row' wrap justify='center'>
-            {renderedProjects?.length && (
+          <StyledBox forwardedAs='ul' direction='row' wrap justify='start'>
+            {renderedProjects?.length ? (
               <li key={renderedProjects?.[0].id} ref={firstCardRef}>
```

Comment:
I left this `ref` here with the first card, but haven't implemented custom focus management. When a user navigates through their pages, I'm not totally sure where the next focus should go. Considering the size of this PR, I'd like to get it merged and then we can do more design adjustments.